src/plugins/vis_type_vega/public/data_model/vega_parser.test.js

```diff
@@ -245,11 +245,11 @@ describe('VegaParser.parseSchema', () => {
 
   test(
     'should not warn on current vega-lite version',
-    check('https://vega.github.io/schema/vega-lite/v4.json', true, 0)
```

Comment:
Should the [default spec](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_vega/public/default.spec.hjson) in the vega plugin be updated to a v5 example, or would that be considered a major-version-upgrade change?

If waiting until the next major version, maybe we should have a tracking issue for that. The current changes could be a bit weird in that the highest support is v5 but everything is still defaulted to v4.

Reply:
That's a great callout - yeah I'd like to update the default in this PR.

Reply:
Updated default spec and test specs in latest commit.